scrapers/LetsDoeIt.yml

```diff
@@ -20,7 +20,7 @@ xPathScrapers:
         selector: //title/text()
         postProcess:
           - replace:
-              - regex: (.+)(?:\s\|.+)
+              - regex: (.+)(?:Exclusive Porn Video)(?:\s\|.+)
                 with: $1
```

Comment:
In cases where the `Exclusive Porn Video` is not present or is a bit different eg dirtycosplay the above breaks the match
I think adding a second regex removing the extra part is safer
```yaml
        postProcess:
          - replace:
              - regex: (.+)(?:\s\|.+)
                with: $1
              - regex: (Exclusive\s(\S+\s)?Porn\sVideo)
                with: ""
```

Reply:
You're right !

Wouldn't `(Exclusive\W+Porn\W+Video)` be better ?

Reply:
If you mean `(Exclusive\W+(\w+\W+)?Porn\W+Video)` that also works
without the extra match group like you have it, it fails in the dirtycosplay case

Reply:
Right !
File updated and Squashed.